benchmark/test_benchmark_mimc.py

```diff
@@ -0,0 +1,43 @@
+from pytest import mark
+from random import randint
+from honeybadgermpc.elliptic_curve import Jubjub
+from honeybadgermpc.progs.mimc import mimc_mpc_batch
+from honeybadgermpc.progs.mixins.share_arithmetic import (
+    BeaverMultiply, BeaverMultiplyArrays, InvertShare, InvertShareArray, DivideShares,
+    DivideShareArrays, Equality)
+
+MIXINS = [
+    BeaverMultiply(),
+    BeaverMultiplyArrays(),
+    InvertShare(),
+    InvertShareArray(),
+    DivideShares(),
+    DivideShareArrays(),
+    Equality()
+]
+
+PREPROCESSING = ['rands', 'triples', 'zeros', 'cubes', 'bits']
+n, t = 4, 1
+k = 300000
+
+TEST_CURVE = Jubjub()
+TEST_FIELD = Jubjub.Field
+TEST_KEY = TEST_FIELD(randint(0, TEST_FIELD.modulus))
+
+
+@mark.asyncio
+@mark.parametrize("batch_size", [10**i for i in range(4)])
+async def test_benchmark_mimc_mpc_batch(batch_size, test_preprocessing, test_runner):
```

Comment:
Oo, do you not have to explicitly make a call to the `benchmark` method? How is this being benchmarked?

Reply:
Refer to an existing benchmark